### ISSUE TITLE
optimize activity bar 

### ISSUE THREAD

**Issue by craigivy** [Microsoft/vscode#29904]
I like having the activity bar, but it creates a lot of wasted space within the IDE.  
- would be nice to decrease the size of the icons, thus decreasing its size
- A minimalistic approach could be to allow it on the status bar

**Comment by bpasero** [comment]
It is possible to hide the activity bar.

**Comment by craigivy** [comment]
Right, but I don't want to hide and unhide it.  Also to be frank, I installed vscode some time ago, and the big activity bar was an immediate turn off.  Now that I've spent some time with it I really like it.  But the activity bar still seems odd in how much space it wastes.  I'm glad I did not let my first impression of the activity bar keep me from using vscode, but I'm concerned others may not feel the same way.  I think developers want the features presented in a minimalistic way in terms of screen real-estate.
